tests/conftest.py

```diff
@@ -86,7 +86,7 @@ def create_project(directory, metadata, *, setup_vcs=True, nested=False):
             git('config', '--local', 'user.email', 'foo@bar.baz')
             git('add', '.')
             git('commit', '-m', 'test')
-            git('tag', '1.2.3')
+            git('tag', '1.2.3', '-m', 'test')
```

Comment:
The old code was causing tests to hang for me, so let me know if this is ok or I should revert it.

Reply:
Let's log a follow up to investigate

Reply:
yes please let's add a todo comment